File: 01_ProgrammingBasics/_02_ConditionalStatements/_02_GreaterNumber.java
package _01_ProgrammingBasics._02_ConditionalStatements;

import java.util.Scanner;

public class _02_GreaterNumber {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read the two integers
        int numberOne = Integer.parseInt(scan.nextLine());
        int numberTwo = Integer.parseInt(scan.nextLine());

        // Print the greater of the two integers
        System.out.println(Math.max(numberOne, numberTwo));
        
        scan.close();
    }
}

File: 23_C#Basics/_02_ConditionalStatements/_02_GreaterNumber/_02_GreaterNumber.cs
using System;

public class _02_GreaterNumber
{
    public static void Main(string[] args)
    {
        // Read the two integers
        int numberOne = int.Parse(Console.ReadLine());
        int numberTwo = int.Parse(Console.ReadLine());

        // Print the greater of the two integers
        Console.WriteLine(Math.Max(numberOne, numberTwo));
    }
}
